jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/ButtonToggleTest.java

```diff
@@ -0,0 +1,129 @@
+package io.github.epam.angular.tests.elements.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.buttonToggleSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class ButtonToggleTest extends TestsInit {
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+    }
+
+    @Test
+    public void basicButtonToggleDisplayedTest() {
+        buttonToggleSection.basicButtonToggleBold.isDisplayed();
+        buttonToggleSection.basicButtonToggleItalic.isDisplayed();
+        buttonToggleSection.basicButtonToggleUnderline.isDisplayed();
+    }
+
+    @Test
+    public void basicButtonToggleEnabledTest() {
+        buttonToggleSection.basicButtonToggleBold.is().enabled();
+        buttonToggleSection.basicButtonToggleItalic.is().enabled();
+        buttonToggleSection.basicButtonToggleUnderline.is().enabled();
+    }
+
+    @Test
+    public void basicBoldButtonToggleTest() {
+        String pressedAttribute = "aria-pressed";
+        String buttonIsPressed = "true";
+        String buttonIsNotPressed = "false";
```

Comment:
The method has variables that are duplicated more than 2 times in an existing class.
It might be better to create 3 fields in a test class.